From cc636133cb87cc590fd653ccd7590506f08641fb Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 12 Dec 2023 16:59:28 +0100
Subject: [PATCH 0101/1695] plat-stm32mp2: embed some extra test facilities

Activate CFG_WERROR and CFG_ENABLE_EMBEDDED_TESTS on stm32mp2 platforms.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 8841801060a32143799616926b87c55866c00267 plat-stm32mp2: embed some extra test facilities]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I5944aa886c4595323ed0670deb31182fa83d672e
---
 core/arch/arm/plat-stm32mp2/conf.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 76dda5797..9f52d51cb 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -49,7 +49,9 @@ CFG_STM32_GPIO ?= y
 CFG_STM32_UART ?= y
 
 # Default enable some test facitilites
+CFG_ENABLE_EMBEDDED_TESTS ?= y
 CFG_WITH_STATS ?= y
+CFG_WERROR ?= y
 
 # Default disable ASLR
 CFG_CORE_ASLR ?= n
-- 
2.47.2

